### PR TITLE
Updating KYC documentation for errors

### PR DESCRIPTION
**Pull request by seguame**
Just a quick update for future usability of these codes for other KYC documents

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -163,9 +163,9 @@ error categories, the last two digits define specific errors.
 * 0333: PIN locked. Too many attempts
 * 0334: Bitso Transfers need either an email, phone_number, or refcode specified
 * 0335: Invalid SPEI recipient name
-* 0336: No data found for CURP
-* 0337: No CURP found for data
-* 0338: Multiple CURPs found for data
+* 0336: No data found for KYC (CURP-México, CUIL-Argentina, CPF-Brasil)
+* 0337: No KYC found for data (CURP-México, CUIL-Argentina, CPF-Brasil)
+* 0338: Multiple KYCs found for data (CURP-México, CUIL-Argentina, CPF-Brasil)
```
Comment (mrduckieduck) [inline source/index.html.md:168]:
```suggestion
* 0336: No data found for KYC (CURP-México, CUIL-Argentina, CPF-Brazil)
* 0337: No KYC found for data (CURP-México, CUIL-Argentina, CPF-Brazil)
* 0338: Multiple KYCs found for data (CURP-México, CUIL-Argentina, CPF-Brazil)
```

File: source/index.html.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ error categories, the last two digits define specific errors.
 * 0344: Invalid BIND recipient name
 * 0345: Quote is expired or invalid
 * 0346: You need to agree to the new terms and condition
-* 0347: Too many attempts to validate CURP
+* 0347: Too many attempts to validate KYC (CURP-México, CUIL-Argentina, CPF-Brasil)
```
Comment (mrduckieduck) [inline source/index.html.md:177]:
```suggestion
* 0347: Too many attempts to validate KYC (CURP-México, CUIL-Argentina, CPF-Brazil)
```

File: source/index.html.md (hunk 1 above)
Comment (seguame) [reply source/index.html.md:168]: I was using Brasil based on how is it written for that country language. Buuuuuuuut I see the point that this is english documentation, in that case I should also remove é from México, just to keep it consistent. WDYT

Comment (mrduckieduck) [reply source/index.html.md:168]: Oh yes  jajaja  👍🏽

**Review by mrduckieduck — APPROVED** [review]

**Review by lrodriguezds — APPROVED** [review]

**Review by viccomx — APPROVED** [review]
LGTM

**Review by domix — APPROVED** [review]
L🐶TM

**Pull request merged**